python/lsst/validate/base/output.py

```diff
@@ -0,0 +1,18 @@
+# See COPYRIGHT file at the top of the source tree.
+from __future__ import absolute_import, division, print_function
+
+import json
+
+from .measurement import MeasurementSet, Measurement
+
+
+def output_measurements(package_name, measurement_dict):
```

Comment:
Yeah. Oddly, this was the original topic for this ticket, but I had to do the metric and measurement stuff to get it to make sense, at which point it faded from relevancy.

We'll probably want to rethink the whole output system, possibly to use the butler metadata system or something, so it plugs into other bits of Tasks trivially.